tools/webdriver/webdriver/bidi/modules/network.py

```diff
@@ -57,11 +57,93 @@ def _add_intercept(self, result: Mapping[str, Any]) -> Any:
         assert result["intercept"] is not None
         return result["intercept"]
 
+    @command
+    def continue_with_auth(
+        self,
+        request: str,
+        action: str,
+        credentials: Optional[Dict[str, str]] = None
```

Comment:
I think that I've mentioned that already earlier during review of this PR. Here we should use a dedicated class for the type similar to the eg. URLPattern ones above.

Reply:
Done (can't use it for the current tests though because we are only testing values which don't match the definition of such a class)